projects/README.md

```diff
@@ -0,0 +1,128 @@
+# 🗂️ Final Challenge
+
+Now that you have learned about the different aspects of building with efficiency in mind, we propose you to pick a final challenge where you can apply all the knowledge you have gained during the course.
+
+In this section, we will present you four different projects with a central theme. Guess it? Efficiency, of course, in a real-world case.
+
+Before you start, check the following pointers:
+
+- The challenges are **unsupervised**. So, we are not providing you with an official solution. You are free to experiment, explore and be as creative as you want. The is no right or wrong way to complete the challenge.
+
+- We recommend you to **select a challenge you are genuinely interested in**. If you are unsure where to begin, check the provided tools and resources. You can start with the easiest level first and then develop the more advanced features.
+
+- Reach out to the **[Discord community](https://discord.com/invite/Tun8YgzxZ9)**. If you are stuck, we will be happy to discuss your ideas and help you get started. If you have completed the challenge, we would be excited to see and share your project.
+
+
+## Pick your challenge
+
+> 💡 Pro Tip
+>
+> Smash your model using Pruna and integrate it in your project.
+> Don't hesitate to check the [Pruna documentation](https://docs.pruna.ai/en/stable/index.html) to get the most out of it.
+
+### 1. **Build your Own Efficient Chatbot**
+
+> 🎯 Goal
+>
+> Design an optimized chatbot powered by an LLM that can answer questions.
+
+#### Description
+
+Build a chatbot that can go beyond basic question answering and respond faster.
+
+- Use an optimized LLM, to which you have applied at least one compression technique.
+- Integrate a database to retrieve and store relevant information.
+- Include a basic UI or API endpoint for interaction
+- **Advanced:** Implement multi-turn interactions with context awareness or multi-modality.
+
+#### Tools and Resources
+
+- Models: [Hugging Face Transformers](https://huggingface.co/docs/transformers/index)
+- Frameworks: [LangChain](https://python.langchain.com/docs/tutorials/chatbot/), [LlamaIndex](https://docs.llamaindex.ai/en/stable/understanding/putting_it_all_together/chatbots/building_a_chatbot/), Haystack
+- Databases: [Weaviate](https://docs.weaviate.io/weaviate/recipes/multi-vector-colipali-rag), FAISS, Chroma, Qdrant
```

Comment:
I think I would recommend going for inmemory databases for each of the providers, not to let people overdo it.